### PR TITLE
[assignmet5] add hometask

### PR DESCRIPTION
**Pull request by OlgaVorokh**
@ariava @yuvdanilova 
Add hometask for review!

### CODE REVIEW

File: assignment5/solver.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    Each line in the file contain one word.
+    All words are ordered by some alphabet that we should recognize.
+    """
+    with open(filename, 'r') as f:
+        words = [line.decode('UTF-8').strip().lower() for line in f]
+    return words
+
+
+class TestSolver(unittest.TestCase):
+    def test_google_test(self):
```
Comment (ariava) [inline assignment5/solver.py:82]: I would inline "words" and "solver" since they are used only once.

**Review by ariava — APPROVED** [review]
Good job! If you can, please add a toplevel comment to your main file that describes the general approach you take in solving the exercise.

**Comment by yuvdanilova** [comment]
Looks great, I like your code style!

**Review by yuvdanilova — APPROVED** [review]
Looks great, I like your code style!

**Pull request merged**